### ISSUE TITLE
Alt text defaults to the image's title if not set on CMS.

### ISSUE THREAD

**Issue by Gazzayeatman** [silverstripe/silverstripe-framework#9099]
## Affected Version

4.3

## Description

More of an improvement than a bug. Can we please add an alt text field to the Image class. At the moment an image's alt text defaults to the image's title if there is no alt text set. There are some cases where the user does no want alt text at all. 

https://www.w3.org/WAI/tutorials/images/decorative/

## Steps to Reproduce

Adding an image in the CMS.

**Comment by robbieaverill** [comment]
cc @silverstripeux

**Comment by clarkepaul** [comment]
Sounds like a duplicate of https://github.com/silverstripe/silverstripe-asset-admin/issues/840

**Comment by brynwhyman** [comment]
Thank you for your report. We've bundled this work and will track it in the following issue: https://github.com/silverstripe/silverstripe-asset-admin/issues/983
